docker-compose.yml

```diff
@@ -141,7 +141,7 @@ services:
       dockerfile: Dockerfile
       target: dev
     restart: no
-    command: bash -c "python manage.py migrate --noinput && python manage.py ensure_superuser --username bfportal --email superuser@bfportal.com --password 1234 &&  python manage.py ensure_initialization && python manage.py runserver 0.0.0.0:8000"
+    command: bash -c "python manage.py migrate --noinput && python manage.py ensure_superuser --username bfportal --email superuser@bfportal.com --password 1234 &&  python manage.py ensure_initialization && python manage.py mock -u 3 -e 10 --clear && python manage.py runserver 0.0.0.0:8000"
```

Comment:
any special reason to create new users and experience every time? As we have set the seed value the data generated will be the same :thinking:

Reply:
https://github.com/battlefield-portal-community/bfportal.gg/blob/68637d82ff6adc8df82a2f834bb15e55c5f86e3f/bfportal/core/management/commands/mock.py#L115

Reply:
Saves executing the command interactively in the started container, right? Otherwise there won't be any data to look at. Same data is also better than no data.